raster/r.surf.gauss/main.c

```diff
@@ -55,9 +57,32 @@ int main(int argc, char *argv[])
     sigma->type = TYPE_DOUBLE;
     sigma->answer = "1.0";
 
+    seed = G_define_option();
+    seed->key = "seed";
+    seed->type = TYPE_INTEGER;
+    seed->required = NO;
+    seed->label = _("Seed for random number generator");
+    seed->description = _("The same seed can be used to obtain same results"
+                          " or random seed can be generated by other means.");
+
     if (G_parser(argc, argv))
         exit(EXIT_FAILURE);
 
+    /****** INITIALISE RANDOM NUMBER GENERATOR ******/
+    if (seed->answer) {
+        seed_value = atol(seed->answer);
+        G_srand48(seed_value);
+        G_verbose_message(_("Read random seed from %s option: %ld"), seed->key,
+                          seed_value);
+    }
+    else {
+        /* default as it used to be */
+        seed_value = G_math_srand_auto();
+        G_warning(_("Warning set option seed. Generated "
+                    "random seed: %ld"),
```

Comment:
I have trouble understanding that sentence. I assume the word "Warning" is superfluous, but the remainder doesn't seem right neither. I think I already wrote a comment on this specific sentence somewhere else.

Reply:
I rewrote the warning to: 

```c
G_warning(_("Autogenerated random seed set to: %ld"), seed_value);
```